### PR TITLE
Renovate README JSON5 instruction

### PR DESCRIPTION
**Pull request by paddyroddy**
Mirror of https://github.com/UCL-ARC/.github/pull/16.

Ignore the failing of the CI, I & @p-j-smith have experienced this locally - have raised an issue https://github.com/AleksaC/hadolint-py/issues/9. If it doesn't get fixed then I suggest we remove `hadolint`.

### CODE REVIEW

File: precommit/mirsg-hooks.yaml [hunk 1]
```diff
@@ -49,7 +49,7 @@ repos:
               }
             }
   - repo: https://github.com/AleksaC/hadolint-py
-    rev: v2.12.0.3
+    rev: v2.12.1b3
```
Comment (paddyroddy) [inline precommit/mirsg-hooks.yaml:52]: This has fixed the issue!

**Comment by paddyroddy** [comment]
`hadolint` issue has been fixed, and have run `pre-commit` so there are some other linting changes

File: precommit/mirsg-hooks.yaml (hunk 1 above)
Comment (p-j-smith) [reply precommit/mirsg-hooks.yaml:52]: awesome!

**Review by p-j-smith — APPROVED** [review]
very nice

**Review by p-j-smith — APPROVED** [review]

**Pull request merged**